### PR TITLE
detail workflow

### PR DESCRIPTION
**Pull request by whyvrafvr**

### CODE REVIEW

File: airdrops/README.md [hunk 1]
```diff
@@ -149,10 +178,62 @@ For a comprehensive understanding, it is recommended to view the following schem
...
+
+In conclusion, by integrating an on-chain transaction airdrop that benefits both the user and their sponsor, PoLN has the potential to create a dynamic and expanding community. 
+
+{% hint style="success" %}
+This approach doesn't just introduce users to the PoLN ecosystem; it embeds them within it, incentivizing further exploration, interaction, and advocacy.
 {% endhint %}
+
+### Propositions
+
+#### NFT Badges
```
[the remark this answers is not in the document]
Comment (s4m5ep10l) [reply airdrops/README.md]: looks like hints are dropped... can I propose to merge "propositions" with "NFT..." and abandon expansion panels ?

[the remark this answers is not in the document]
Comment (whyvrafvr) [reply airdrops/README.md]: sure

[the remark this answers is not in the document]
Comment (s4m5ep10l) [reply airdrops/README.md]: so ?

[the remark this answers is not in the document]
Comment (whyvrafvr) [reply airdrops/README.md]: that's fine 🆗

[the remark this answers is not in the document]
Comment (whyvrafvr) [reply airdrops/README.md]: you can merge @s4m5ep10l

**Comment by s4m5ep10l** [comment]
I can't rebase from github website. Can you do it ?

**Comment by whyvrafvr** [comment]
> I can't rebase from github website. Can you do it ?

LOL sure.

**Comment by s4m5ep10l** [comment]
last but not least, you might remove the pic icon and (WIP) status buddy...

**Pull request merged**